docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.3"
+services:
+  app:
+    image: todo-app:dev
```

Comment:
Using the image tag is fine, but if you want to combine the build step here as well we can do that with e.g.
```
    build: 
      context: .
      target: development
```